_includes/getting-started.html

```diff
@@ -63,44 +98,84 @@ <h3 style="margin-left:1em;">Kubernetes user</h3 style="margin-left:1.5em;">
             <!-- Kubernetes - Helm -->
             <h4 style="font-size: 1.5em;">Using Helm</h4>
             <p>Helm v3 - run the following:</p>
-            <div class="highlight-code highlight-source-shell">
-              <pre><code>
-  $ git clone https://github.com/layer5io/meshery.git; cd meshery
-  $ kubectl create namespace meshery
-  $ helm install meshery --namespace meshery install/kubernetes/helm/meshery
-              </code></pre>
+            <div id="start-brew" class="highlight-code highlight-source-shell" style="margin-left:1.8em;">
+              <div style="width:auto; height:60px; padding:0px; position:relative;">
+                <pre><code> $ git clone https://github.com/layer5io/meshery.git; cd meshery<br /> $ kubectl create namespace meshery<br /> $ helm install meshery --namespace meshery install/kubernetes/helm/meshery<br /> 
+                </code></pre>
+                <!-- copy to clipboard -->
+                <a class="btn waves-effect waves-dark white-text darken-2 l5-light-blue z-depth-4"
+                style="position: absolute;top:-14px; right: -1px; border: 0; 
+                    cursor: pointer; background:#00a4ca; width:60px; height:75px; margin-bottom:35px; 
+                    border-radius: 2px; margin:0px 0px 880px 0px;"
+                  data-clipboard-target="#start-brew"
+                  data-clipboard-text="brew tap layer5io/tap
+                  brew install mesheryctl
+                  mesheryctl start">
+                  <img src="/images/buttons/clippy.svg" style="width:20px; margin: 0; position: absolute; top: 50%; left: 54%; transform: translate(-50%, -50%);" />
+                </a>
             </div>
+          </div>
 
             <p>Helm v2 - run the following:</p>
-              <div class="highlight-code highlight-source-shell">
-              <pre><code>
-  $ git clone https://github.com/layer5io/meshery.git; cd meshery
-  $ kubectl create namespace meshery
-  $ helm template meshery --namespace meshery install/kubernetes/helm/meshery
-              </code></pre>
+            <div id="start-brew" class="highlight-code highlight-source-shell" style="margin-left:1.8em;">
```

Comment:
The div id must be unique. Should be something like "start-helm-v2". This id needs to match the `data-clipboard-target` value on line 129.